docs/how-to/debugging_env.rst

```diff
@@ -0,0 +1,94 @@
+.. list-table::
+    :header-rows: 1
+    :widths: 35,14,51
+
+    * - **Environment variable**
+      - **Default value**
+      - **Value**
+
+    * - | ``AMD_LOG_LEVEL``
+        | Enables HIP log on various level.
+      - ``0``
+      - | 0: Disable log.
+        | 1: Enables log on error level.
```

Comment:
I think the word level is redundant here. Something like Enable Error Logs or Enable Warning logs etc should be fine.

Reply:
Fixed.